src/components/enhanced-table/enhanced-table-pagination/EnhancedTablePagination.jsx

```diff
@@ -70,13 +71,13 @@ const EnhancedTablePagination = ({ pagination }) => {
           type='number'
           value={pageInput}
         />
-        <Button
+        <AppButton
           onClick={() => handlePageSubmit(pageCount)}
           sx={styles.btn}
-          variant='outlined'
+          variant={ButtonVariantEnum.Contained}
```

Comment:
Isn't `contained` not the same as `outlined`?

Reply:
`Outlined` button variant was deleted from theme completely (discussed with designer). I decided to use `contained` as default for now and also this component is used on admin dashboard and it is not our priority right now.